syncd/VendorSai.cpp

```diff
@@ -113,6 +113,10 @@ sai_status_t VendorSai::uninitialize(void)
     SWSS_LOG_ENTER();
     VENDOR_CHECK_API_INITIALIZED();
 
+#ifdef MDIO_ACCESS_USE_NPU
+    MdioIpcServer::clearSwitchMdioApi();
+#endif
+
```

Comment:
please remove any of this code from VendorSai, this should be standalone module

Reply:
virtual functions mdioRegRead/mdioRegWrite are declared in SaiInterfaces class and are overridden in VendorSai class. MdioIpcServer::clearSwitchMdioApi is no longer needed.